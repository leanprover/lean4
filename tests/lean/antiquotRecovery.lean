syntax "foo " (" bar " ident)? : tactic

macro_rules
  -- should not complain about `$id` not being a valid command and create a snapshot for it
  | `(tactic| foo $[bar $id]) => sorry
