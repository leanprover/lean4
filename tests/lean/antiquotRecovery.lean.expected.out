antiquotRecovery.lean:5:18: error: expected ')'
